docs/docs/smart-contracts/example/contract.md

```diff
@@ -21,15 +21,15 @@ authors = ["Brushfam <dominik.krizo@727.ventures>"]
 edition = "2021"
 
 [dependencies]
-ink = { version = "4.1.0-beta", default-features = false }
+ink = { version = "4.2.1", default-features = false}
 scale = { package = "parity-scale-codec", version = "3", default-features = false, features = ["derive"] }
 scale-info = { version = "2.6", default-features = false, features = ["derive"], optional = true }
 
 # These dependencies
 shares_contract = { path = "../shares", default-features = false, features = ["ink-as-dependency"]  }
 loan_contract = { path = "../loan", default-features = false, features = ["ink-as-dependency"]  }
 lending_project = { path = "../..", default-features = false }
-openbrush = { version = "~3.1.1", default-features = false, features = ["pausable", "access_control"] }
+openbrush = { git = "https://github.com/Brushfam/openbrush-contracts", branch = "feature/stable-rust", default-features = false }
```

Comment:
it's better to change feature/stable-rust here to develop or main

Reply:
for now I used this branch everywhere so docs are aligned with code when testing